cmd/rmr.go

```diff
@@ -44,16 +43,22 @@ $ juicefs rmr /mnt/jfs/foo`,
 	}
 }
 
-func openController(path string) *os.File {
-	f, err := os.OpenFile(filepath.Join(path, ".control"), os.O_RDWR, 0)
-	if err != nil && !os.IsNotExist(err) && !errors.Is(err, syscall.ENOTDIR) {
-		logger.Errorf("%s", err)
-		return nil
+func openController(mp string) *os.File {
+	st, err := os.Stat(mp)
+	if err != nil {
+		logger.Fatal(err)
+	}
+	if !st.IsDir() {
+		mp = filepath.Dir(mp)
 	}
-	if err != nil && path != "/" {
-		return openController(filepath.Dir(path))
+	for ; mp != "/"; mp = filepath.Dir(mp) {
```

Comment:
With the change in vfs we can access .control under any directory in juicefs, so we don't need the for loop any more.

Reply:
This loop is used to find out the mount point to pass relative pass into client.

Reply:
mp is not returned in this func

Reply:
For backward compatibility, new version should work with existing mount point.